test/jdk/tools/jlink/plugins/CDSPluginTest.java

```diff
@@ -102,4 +121,20 @@ public static void main(String[] args) throws Throwable {
            out.shouldHaveExitValue(1);
        }
     }
+
+    private static void copyModuleFiles(Path jdkRoot, Path targetDir, String[] modules)
```

Comment:
This method copies "packaged modules" so I think should be renamed to copyPackagedModules.